LottoDiary/LottoDiary/Sources/Present/ViewController.swift

```diff
@@ -28,6 +36,14 @@ class ViewController: UIViewController {
             make.size.equalTo(200)
         }
         // Do any additional setup after loading the view.
+        
+        view.addSubview(codeReviewLabel)
+        codeReviewLabel.snp.makeConstraints { make in
+            make.top.equalTo(testLabel.snp.bottom).inset(20)
+            make.leading.trailing.equalToSuperview().offset(30)
+            make.width.equalToSuperview()
+            make.height.equalTo(50)
+        }
```

Comment:
```suggestion
        codeReviewLabel.snp.makeConstraints { make in
            make.top.equalTo(testLabel.snp.bottom).inset(20)
            make.leading.trailing.equalToSuperview().offset(30)
            make.width.equalToSuperview()
            make.height.equalTo(70)
        }
```
codeReviewLabel의 height는 50보다 70이 더 적당할 것 같습니다.

Reply:
굿이요